### PR TITLE
Writes unit tests to ZohoAPI class

### LINKED ISSUE #63: Add unit tests to Zoho project
I wanna see such bage on xola Readme page of GitHub

### PR DESCRIPTION
**Pull request by mytrov**
### What does it do?
- Adds tests for ZohoAPI methods
- Creates package `tests` with __init__.py file
- Adds small fix for zoho_api.py for the import setup file

### Checklist
- [x] Pipeline set to `Ready To Deploy` or `Review/QA`
- [x] Pull request is prepared: Added to sprint, added yourself as assignee, Zenhub task is linked, reviewers are added
- [x] PR is not very big (about 200 lines of change)
- [x] Doesn't create any new `FIXME` or `TODO` comments. If not, please explain or create issues to resolve it.
- [x] Doesn’t deliver half-baked changes
- [x] Tested manually or with a new unit test
- [x] Follows best practice rules

### CODE REVIEW

**Comment by BearVictor** [comment]
I try to start pytest for a different level of the directory, but always have the error.
image

File: apis/zoho_api.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from requests import RequestException
 import zcrmsdk as zoho_crm
 import pytz
-from setup import CONFIG_FILE
+from ..setup import CONFIG_FILE
```
Comment (iramykytyn) [inline apis/zoho_api.py:13]: why `..`?

**Review by iramykytyn — APPROVED** [review]
Tests look good.

Is that issue Victor mentioned now fixed?

File: apis/zoho_api.py (hunk 1 above)
Comment (mytrov) [reply apis/zoho_api.py:13]: It's relative path
Because of it we can import higher level files

**Pull request merged**